File: frontp/src/models/imc.ts
import { Aluno } from "./aluno";

export interface Imc {
  ImcId?: string;
  Imc: string;
  classificacao: string;
  peso: string;
  altura?: string;
  alunoId: string;
  aluno?: Aluno;
  criadoEm?: string;
}

File: api/api/Models/Imc.cs
public class Imc
{
    public int? id { get; set; }
    public double? Im { get; set; }
    public string? Classificacao { get; set; }
    public double? Altura { get; set; }
    public double? Peso { get; set; }
    public int AlunoId { get; set; }
    public Aluno? Aluno { get; set; }
    public DateTime? CriandoEm { get; set; } = DateTime.Now;
}
